document/core/text/modules.rst

```diff
@@ -425,7 +427,8 @@ Globals can be defined as :ref:`imports <text-import>` or :ref:`exports <text-ex
        (\iff \Tid^? \neq \epsilon \wedge \Tid' = \Tid^? \vee \Tid^? = \epsilon \wedge \Tid' \idfresh) \\
    \end{array}
 
-Only the export abbreviation can be repeated, with ":math:`\dots`". This means expansion can result in any number of exports followed by up to one import.
+   The latter abbreviation can be applied repeatedly, if ":math:`\dots`" contains additional export clauses.
+   Consequently, a function definition can contain any number of exports, followed by either an import or a global type.
```

Comment:
not sure "global type" is the best wording here, so if you have a better one let me know!

Reply:
The type is present in either case, the difference would be the initialiser after the type. But see my suggestion to drop that part.